client/tests/phpunit/Command/CheckCSVUploadedCommandTest.php

```diff
@@ -0,0 +1,427 @@
+            ->shouldBeCalled();
+        $slackClient->chatPostMessage([
+                                          'username' => 'opg_response',
+                                          'channel' => 'opg-digideps-team',
+                                          'text' => 'The Sirius Lay CSV has not been uploaded within the past 24 hours',
+                                      ])
+            ->shouldBeCalled();
+        $slackClient->chatPostMessage([
+                                          'username' => 'opg_response',
+                                          'channel' => 'opg-digideps-team',
+                                          'text' => 'The CasRec Prof CSV has not been uploaded within the past 24 hours',
+                                      ])
+            ->shouldBeCalled();
+        $slackClient->chatPostMessage([
+                                          'username' => 'opg_response',
+                                          'channel' => 'opg-digideps-team',
+                                          'text' => 'The CasRec PA CSV has not been uploaded within the past 24 hours',
+                                      ])
+            ->shouldBeCalled();
+
+        $this->slackClientFactory->createClient($this->slackSecret)
+            ->shouldBeCalled()
+            ->willReturn($slackClient->reveal());
+
+        $result = $this->commandTester->execute([]);
+
+        $this->assertEquals(0, $result, sprintf('Expected command to return 0, got %d', $result));
+    }
+
+    /**
+     * @test
+     */
+    public function executeOnNonBankHolidaysWhenACasRecLayCSVHaveNotBeenUploadedSlackIsPostedTo()
+    {
+        $this->todayIsABankHoliday(false);
+        $this->aCsvUploadedEventExists(true, [
+            CheckCSVUploadedCommand::SIRIUS_LAY_CSV,
+            CheckCSVUploadedCommand::CASREC_PROF_CSV,
+            CheckCSVUploadedCommand::CASREC_PA_CSV,
+        ]);
+
+        $this->secretManagerService->getSecret('opg-response-slack-token')
+            ->shouldBeCalled()
+            ->willReturn($this->slackSecret);
+
+        $slackClient = self::prophesize(Client::class);
+        $slackClient->chatPostMessage([
+                                          'username' => 'opg_response',
+                                          'channel' => 'opg-digideps-team',
+                                          'text' => 'The CasRec Lay CSV has not been uploaded within the past 24 hours',
+                                      ])
+            ->shouldBeCalled();
+        $this->slackClientFactory->createClient($this->slackSecret)
+            ->shouldBeCalled()
+            ->willReturn($slackClient->reveal());
+
+        $result = $this->commandTester->execute([]);
+
+        $this->assertEquals(0, $result, sprintf('Expected command to return 0, got %d', $result));
+    }
+
+    /**
+     * @test
+     */
+    public function executeOnNonBankHolidaysWhereOnlyNonCSVEventExistsSlackIsPostedTo()
```

Comment:
Yeah this test case has a really confusing name.
Essentially I wanted to have a test case for when there are events in the log stream, but they don't match any CSV upload events. It's a very unlikely scenario to be honest, but I think it was best to ensure that the command still works in this case.
In the scenario although there are events in the log stream, because none of them match the CSV upload events, we will alert for all CSVs.